docs/installation/upgrading.rst

```diff
@@ -39,3 +39,11 @@ Notable changes in 4.1:
 * Configuration: Advanced KSQL users can configure the Kafka Streams and Kafka producer/consumer client settings used
   by KSQL.  This is achieved by using prefixes for the respective configuration settings.
   See :ref:`ksql-param-reference` as well as :ref:`ksql-server-config` and :ref:`install_cli-config` for details.
+
+Upgrading from KSQL 4.1 and 5.0 (Developer Preview) to KSQL 5.1
```

Comment:
This should just be "Upgrading from KSQL versions 5.0.0 and below". It would make the note more relevant beyond version 5.1.

Also the `rst` syntax requires the header underline to match the line length explicitly.